modules/openapi-generator/src/main/resources/JavaJaxRS/spec/pojo.mustache

```diff
@@ -46,11 +46,33 @@ public class {{classname}} {{#parent}}extends {{{.}}}{{/parent}} {{#vendorExtens
     {{/vars}}
   }
 
-  public {{classname}}() {
-  }
   {{/additionalProperties}}
   {{/generateBuilders}}
+  public {{classname}}() {
```

Comment:
I'm not familiar with the JavaJaxRS code, but should this have a ^hasRequired around it? Or are both constructors valid?

Reply:
We need a default constructor for cases where there is a constructor with arguments in order not to break existing code.
This can happen with hasRequired (new case) or with generateBuilders (existing case), but as far as I know it is not possible to do a OR in mustache.
So I took the simple route and put the default constructor unconditionally, thinking that if in the future a new case with another constructor with arguments arises, it will not be forgotten.